src/components/Layouts/ReportLayout.less

```diff
@@ -163,6 +163,10 @@ h1 {
         }
 
         @media print {
```

Comment:
what's the point of making a print media query here when it's the main and only use case for the UI?

Reply:
@yanyo87 can you add info on that?

Reply:
from my point of view, 
the purpose of this media query is to hide scrollbars when printing,
but to show them when developing